docs/UserGuide.md

```diff
@@ -294,7 +307,9 @@ Examples:
 
 ### Search tasks with tag: `findTag`
 
-Searches for tasks with an associated tag. Note that tags are case-sensitive.
+Searches for tasks with an associated tag. Tag is case-insensitive for your convenience.
+
+One Tag allowed. Tag provided mut be alphanumerical.
```

Comment:
Noticed a spelling mistake `mut` instead of `must`.

Reply:
oops! will change now